components/omega/test/testdummy.cpp

```diff
@@ -1,3 +1,4 @@
+#include "logging.h"
 #include <iostream>
 
 using namespace std;
```

Comment:
llvm style also discourages this and prefers explicit std:: throughout

Reply:
OK